pkg/controller/controller.go

```diff
@@ -64,6 +65,10 @@ func New(cfg Config) *Controller {
 func (c *Controller) handleClusterEvent(event *Event) error {
 	clus := event.Object
 
+	if !c.managed(clus) {
+		return fmt.Errorf("cluster (%s) isn't managed", clus.Name)
```

Comment:
This will introduce unnecessary errors/warnings in the log for unmanaged EtcdCluster CRs. Should just ignore it.

Reply:
Make this func return `(ignored bool, err error)` ?

Reply:
@hongchaodeng tell me if my last commit is ok for you.
I will rebase if ok.